code/game/objects/structures/cannons/mounted_guns/mounted_gun.dm

```diff
@@ -9,20 +9,37 @@
 	icon = 'icons/obj/weapons/cannons.dmi'
 	icon_state = "falconet_patina"
 	var/icon_state_base = "falconet_patina"
+	var/icon_state_loaded = "falconet_patina"
 	var/icon_state_fire = "falconet_patina_fire"
 	max_integrity = 300
 	///whether the cannon can be unwrenched from the ground. Anchorable_cannon equivalent.
 	var/anchorable_gun = TRUE
+	/// does this thing need ammo at all or does it just make ammo?
+	var/uses_ammo= TRUE
 	///Max shots per firing of the gun.
 	var/max_shots_per_fire = 1
+	///Is there a do_after when loading the gun?
+	var/has_loading_delay = FALSE
+	///Delay it takes to load the gun. Set to 0 if none.
+	var/load_delay = 0
+	///Message displayed when loading gun
+	var/loading_message = "You Loaded The Gun."
```

Comment:
```suggestion
	var/loading_message = "gun loaded"
```
This is a balloon alert, it should be as brief as possible, uncapitalised, and not use punctuation (except ! for failures)